tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml

```diff
@@ -167,8 +166,9 @@ stages:
                   --parallel \
                   --enable_lto \
                   --build_wheel \
+                  --use_openmp \
                   --enable_onnx_tests \
-                  --wheel_name_suffix=noopenmp \
+                  --wheel_name_suffix=openmp \
```

Comment:
>-wheel_name_suffix=openmp [](start = 19, length = 25)

So in some cases you add openmp.  I think the fact needs to be reflected in the PR description.

Reply:
Thanks! How about now?